pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+license_file = "LICENSE"
+description-file = "README.md"
+
+
+[tool.pytest.ini_options]
+addopts = '-m "not conda"'
+markers = [
+    "conda: test related to conda forge distribution"
+]
```

Comment:
@MichalChromcak @skzhang1 could you help review this part as it seems related to conda?